throttle/contract_tx_limiter_middleware.go

```diff
@@ -11,12 +14,57 @@ import (
 	udw "github.com/loomnetwork/loomchain/builtin/plugins/user_deployer_whitelist"
 	"github.com/loomnetwork/loomchain/vm"
 	"github.com/pkg/errors"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
 )
 
 var (
 	ErrTxLimitReached = errors.New("tx limit reached, try again later")
 )
 
+var (
+	tierMapLoadLatency                   metrics.Histogram
+	contractTierMapLoadLatency           metrics.Histogram
+	contractStatsMapLoadLatency          metrics.Histogram
+	contractTxMiddlewareExecutionLatency metrics.Histogram
+)
+
+func init() {
+	fieldKeys := []string{"method", "error"}
+
+	tierMapLoadLatency = kitprometheus.NewSummaryFrom(stdprometheus.SummaryOpts{
+		Namespace:  "loomchain",
+		Subsystem:  "contract_tx_limiter_middleware",
+		Name:       "tier_map_load_latency_microseconds",
```

Comment:
Remove `_microseconds` from all the names, it's actually seconds, but the units shouldn't be part of the name anyway.

Reply:
Done.